packages/grid/x-data-grid-generator/src/renderer/renderTotalPrice.tsx

```diff
@@ -1,27 +1,24 @@
 import * as React from 'react';
 import clsx from 'clsx';
-import { alpha, createTheme } from '@mui/material/styles';
-import { createStyles, makeStyles } from '@mui/styles';
+import { alpha, styled } from '@mui/material/styles';
 import { GridCellParams } from '@mui/x-data-grid';
 
-const defaultTheme = createTheme();
-const useStyles = makeStyles(
-  (theme) =>
-    createStyles({
-      root: {
-        width: '100%',
-        paddingRight: 8,
-        fontVariantNumeric: 'tabular-nums',
-      },
-      good: {
-        backgroundColor: alpha(theme.palette.success.main, 0.3),
-      },
-      bad: {
-        backgroundColor: alpha(theme.palette.error.main, 0.3),
-      },
-    }),
-  { defaultTheme },
-);
+const Value = styled('div')(({ theme }) => ({
+  width: '100%',
+  height: '100%',
```

Comment:
The added `height` is to fix a bug created by #3446. It was also added in edit components.

![image](https://user-images.githubusercontent.com/42154031/148628655-4c50051c-7b3c-43dd-bd5b-fb65480c3a7d.png)

![image](https://user-images.githubusercontent.com/42154031/148628665-25371ac0-d8f3-46ca-9248-d7ca24e78f00.png)